### PR TITLE
Add Rouge support

### PR DESCRIPTION
**Pull request by jgoizueta**
Pygments is fine, but I like using fenced code blocks rather than highlight liquid tags. Fenced blocks are nicely highlighted by Atom in the source code, and can be rendered by kramdown+rouge using the same Pygment styles.

### CODE REVIEW

**Comment by thomasjo** [comment]
See https://github.com/github/pages-gem/pull/79.

**Comment by jgoizueta** [comment]
OK, I guess I should look for a solution on the Jekyll/Markdown side that relies on Pygments for the time being.

Hope to have news about PrettyLights soon.

File: lib/github-pages.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class GitHubPages
-  VERSION = 38
+  VERSION = 39
```
Comment (parkr) [inline lib/github-pages.rb]: :question:

Comment (jgoizueta) [inline lib/github-pages.rb]: Oops, I shouldn't have committed that

Comment (Stargator) [inline lib/github-pages.rb]: You can close this pull request, create a new branch that only has the changes you desire and then do another pull request. I did that for a few requests that had unintended changes.
